### PR TITLE
Fix typos and dead links. Add TODO for dead links that weren't a quick fix.

### PR DESCRIPTION
**Pull request by rstorr**

### CODE REVIEW

File: docs/concepts/examples.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ query:
 
 Lexicons get data into the CodeLingo Platform and provide a list of facts to query that data. In the above example, the git Lexicon finds and clones the "myrepo" repository from the "myvcsgithost.com" VCS host. The "myrepo" repository must be publicly accessible for the git lexicon to access it.
 
-The CodeLingo Platform can be queried directly with the the `$ lingo search` command or via [Functions](/concepts/flows.md) which use queries stored in Tenets.
+The CodeLingo Platform can be queried directly with the `$ lingo search` command or via [Functions](flows.md) which use queries stored in Tenets.
```
Comment (emersonwood) [inline docs/concepts/examples.md]: Nice, the command is also outdated / incorrect here and should be `$ lingo run search`.

File: docs/concepts/flows.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ query: |
 
 By decorating the `go.func_decl` fact with `@review.comment` the review Flow's comment Function extracts the file and line from the query result, to be used by the review Flow to comment on that line.
 
+<!-- TODO fix HTML link. -->
```
Comment (BlakeMScurr) [inline docs/concepts/flows.md]: Where does it point to now? Is there a quick hack we can do?

Comment (BlakeMScurr) [reply docs/concepts/flows.md]: Looks like it's not broken on the deployed site. Do you mean it would just be nicer to have this in markdown?

Comment (rstorr) [reply docs/concepts/flows.md]: MkDocs ignores raw HTML. The hack is just to link to the full url of tenets (https://www.codelingo.io/docs/concepts/tenets/) I'll do that...

Comment (rstorr) [reply docs/concepts/flows.md]: oh.. I'm getting 404 on localhost. I'll remove the todo.

**Pull request merged**